tests/unit_tests/test_cloud.py

```diff
@@ -12,3 +13,23 @@ def test_cloud_get_reservations_available_resources(specific_reservations,
     available_resources = Cloud().get_reservations_available_resources(
         "instance_type", "region", "zone", specific_reservations)
     assert available_resources == expected
+
+
+class TestCheckClusterNameIsValid:
+
+    def test_check(self):
+        Cloud.check_cluster_name_is_valid("lora")
```

Comment:
We probably want to refactor the `Cloud.check_cluster_name_is_valid` to `common_utils.check_cluster_name_is_valid`. Reason: the validity of a `cluster_name` is no longer cloud dependent but a general check. It might be better to have it in `common_utils` instead.

In that case, we can also move these tests to `test_common_utils` as well.